src/engine/Drawing/Sprite.ts

```diff
@@ -7,12 +7,14 @@ import { Vector } from '../Algebra';
 import { Logger } from '../Util/Log';
 import { clamp } from '../Util/Util';
 import { Configurable } from '../Configurable';
+import { obsolete } from '../Util/Decorators';
 
 /**
  * @hidden
+ * @deprecated Use [["Graphics/Sprite".Sprite]]
  */
 export class SpriteImpl implements Drawable {
-  private _texture: Texture;
+  public _texture: Texture;
```

Comment:
```suggestion
  public texture: Texture;
```

Changing accessibility maybe need to update name?